src/test/groovy/org/ods/doc/gen/leva/doc/fixture/LevaDocWiremock.groovy

```diff
@@ -35,6 +36,7 @@ class LevaDocWiremock {
         this.environment = environment
     }
 
+    @WithReadLock
```

Comment:
I don't understand why are you using this annotation without use @WithWriteLock in any place

Reply:
removed